src/libraries/System.Text.RegularExpressions/src/System/Text/RegularExpressions/RegexFindOptimizations.cs

```diff
@@ -434,6 +436,7 @@ public bool TryFindNextStartingPositionRightToLeft(ReadOnlySpan<char> textSpan,
             }
         }
 
+#if SYSTEM_TEXT_REGULAREXPRESSIONS
```

Comment:
What is this for? Is the intention for the source generator or tests not to have this method?

Reply:
The file is built into both S.T.RegularExpressions.dll and the generator, the latter of which builds for netstandard2.0 and doesn't have some of the APIs used in this function.  But this function is also only about performing matches at execution time, so it's not relevant to the generator, anyway.